

export const registerUser = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).json({success: false, msg: error, errors: []})
    }
};

export const logInUser = async (req, res) => {
    try {

    } catch (error) {
        res.status(500).json({success: false, msg: error, errors: []})
    }
};

